src/AzDoBridge/Helpers/AADAuthenticator.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Protocols;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Claims;
+using Microsoft.Extensions.Logging;
+using System.IdentityModel.Tokens.Jwt;
+using System.Threading;
+using Newtonsoft.Json;
+using System.Net.Http.Headers;
+
+namespace AzDoBridge.Helpers
+{
+    public static class AADAuthenticator
```

Comment:
Please do not use static classes. This is not supported by Dependency Injection.
We should build an extendable and replaceable authentication based on interfaces and dependency-injectionable